CHANGELOG.md

```diff
@@ -24,6 +24,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - `Binarizer` preprocessing model
 - Preprocessing models test runner
 ### Changed
+- `Command` enum class in `transporter.py`
+- `SerilaizatoinErrorTypes` error type in `serialize_exception.py`
```

Comment:
- `SerilaizatoinErrorTypes` is an `enum` class same as the `Command` class so we can use the same format as the line 27th.
- I think it's better to use the fixed typo one, I mean `SerializationErrorTypes`